src/modules/verifiable-credentials/storage-client.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  PresentationDefinition,
+  VerifiableCredential,
+  CredentialSubject,
+} from '@ew-did-registry/credentials-interface';
+import { AxiosInstance } from 'axios';
+import { CacheClient } from '../cacheClient';
+
+export default class VCStorageClient {
+  private _http: AxiosInstance;
+  private _baseUrl = 'vc';
```

Comment:
We will be using `vp` and `webnode` prefix, so I think we can remove  this parameter